docs/fundamentals/toc.yml

```diff
@@ -2099,6 +2099,47 @@ items:
     href: ../core/extensions/generic-host.md
   - name: HTTP with .NET
     href: ../core/extensions/http-client.md
+  - name: Globalization and localization
+    href: ../core/extensions/globalization-and-localization.md
```

Comment:
Can you move this href under items and name it "Overview"?

Reply:
Ah, good catch - meant to do that.